commons/src/main/scala/dagr/commons/CommonsDef.scala

```diff
@@ -62,4 +63,35 @@ object DagrDef {
 
   /** The type of identifier used to uniquely identify tasks tracked by the execution system. */
   type TaskId = BigInt
+
```

Comment:
Good catch on the `TaskId`.  I do want to keep the `PathTo*` in here, since I will want o use them for `JeanLuc`.  How about a second def (`BfxPathDef` or something)?